src/radix_connect/mobile/client.rs

```diff
@@ -70,9 +70,7 @@ impl RadixConnectMobile {
                 },
             );
         }
-        let callback_path = dapp_definitions
-            .callback_path
-            .map_or(RCMCallbackPath::default(), RCMCallbackPath::new);
+        let callback_path = dapp_definitions.callback_path.unwrap_or_default();
```

Comment:
the code and comment in the end of this function:

```rust
       // 5. TODO: use the actual dapp callback path
        Ok(Url::from_str("https://example.com").unwrap())
```

hmm... should we not do:

```rust
Ok(callback_path)
```
???

Or we we have two different callback paths?

Reply:
this is code Ghenadie wrote, I just added parsing callback_path and storing it. But yes, we will need to combine `origin` and `callback_path` together as final result of this method

Reply:
@jakub-rdx you can use the callback_path now to construct the redirect URL now that we have it.